Vagrantfile

```diff
@@ -0,0 +1,38 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+@script = <<SCRIPT
+apt-get update
+apt-get install -y apache2 git curl php5-cli php5 libapache2-mod-php5
+echo "
+<VirtualHost *:80>
+    ServerName zf2-tutorial.localhost
+    DocumentRoot /var/www/zf/public
+    <Directory /var/www/zf/public>
+        DirectoryIndex index.php
+        AllowOverride All
+        Order allow,deny
+        Allow from all
+    </Directory>
+</VirtualHost>
+" > /etc/apache2/sites-available/000-default.conf
```

Comment:
I preffer if use his own file then enable the vhost and disable the default one.